scripts/globals/mobskills/severing_fang.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------
+--  Severing Fang
+--
+--  Description: Bites at all targets in front. Additional effect: Defense Down
+--  Type: Physical
+--  Utsusemi/Blink absorb: 2 shadows
+--  Range: Front arc (cone)
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+  -- Do not use this weapon skill on targets behind. Sub-Zero Smash
+  -- should trigger in this case.
+  if target:isBehind(mob) then
+    return 1
+  end
```

Comment:
Work looks really reasonable across all PRs, the only thing I'd suggest is a bit of styling changes.  4-space indents as a standard is what we're holding to right now. 👍

Reply:
Ah, force of habit from work (we use 2 spaces).

Please nitpick any inconsistencies because I may miss a few...

Will fix.